### ISSUE TITLE
Instagram ads aren't blocked (both feed and story ones)

### ISSUE THREAD

**Issue by MarcAnt01** [AdAway/AdAway#1071]
I don't know whether AdAway can do it, but I've opened an issue, because I don't know where else I could ask

**Comment by patrickdrd** [comment]
nope, sorry mate but adaway cannot block instagram ads, neither twitter ads, nor youtube ads

**Comment by MarcAnt01** [comment]
I use twitter via browser and use Vanced for Youtube, but no Instagram client is ad free, that's why I asked😢

**Comment by patrickdrd** [comment]
you might have some luck if you're rooted and use xposed with xinsta,
but that's ambivalent

**Comment by MarcAnt01** [comment]
Xposed is shit, it makes my device last half day and lag it. Some time ago I have tried for curiosity and anyway XInsta does block just feed ads and not story ones, which are the most annoying
